### ISSUE TITLE
Can't make basic connection to Confluence Page

### ISSUE THREAD

**Issue by chrisg423** [atlassian-api/atlassian-python-api#1070]
This is an issue I am having trying to use the Atlassian Python API's Confluence module to edit a page. So far, all I have in my code is the call to connect to the page using:

`conf = Confluence(url=conf_site, username=conf_user, password=conf_pass, cloud=True)`
(obviously I have real values set for the url, username, and password)

and then a call to try and get the status of a page:

`status = conf.get_page_properties(page_id=x)`
(again, obviously with a real value filled in for x)

and then I try to print that status. However, I get a bunch of traceback errors that look like this:

<img width="1680" alt="Screen Shot 2022-11-04 at 3 49 10 PM" src="https://user-images.githubusercontent.com/44785108/200063456-ab636bf7-030f-40fc-bb7e-aa4ac4262253.png">

I have tried this both on a Windows PC as well as a Mac. I get the same sort of traceback errors both times. Please let me know if you have seen anything like this before or if you can point me in the right direction. Thank you.

**Comment by gonchik** [comment]
Hi!
A I see you're using on prem Confluence could you try the next parameter
```
conf = Confluence(url=conf_site, username=conf_user, password=conf_pass, cloud=False)
```

**Comment by chrisg423** [comment]
> Hi! A I see you're using on prem Confluence could you try the next parameter
> 
> ```
> conf = Confluence(url=conf_site, username=conf_user, password=conf_pass, cloud=False)
> ```

Did that, still nothing. Got the exact same traceback errors.

**Comment by gonchik** [comment]
Just curious can you make a telnet request? 

```telnet confluence.example.com 443```

**Comment by chrisg423** [comment]
> Just curious can you make a telnet request?
> 
> `telnet confluence.example.com 443`

Excuse my ignorance with Telnet, but I just installed and tried to do that and I am getting 
nodename nor servname provided, or not known

Any idea how to deal with that?

**Comment by gonchik** [comment]
@chrisg423  so you need to check your DNS records. 
1. Is this on-prem installation ? if so. you can put record into /etc/hosts or ping system admins.

**Comment by chrisg423** [comment]
> @chrisg423 so you need to check your DNS records.
> 
> 1. Is this on-prem installation ? if so. you can put record into /etc/hosts or ping system admins.

Yes, this is on-prem. I did just realize we go through a proxy server though. Is there anything special I'll have to do for that? I did not see anything noted in the documentation.

**Comment by chrisg423** [comment]
Hi, still wondering if there is anything special I would have to do to bypass a proxy server. Please let me know if you can. Thank you.

**Comment by gonchik** [comment]
You can use the cookies parameters to go through proxy server. 
just copy from your browser the cookies and put into script

**Comment by chrisg423** [comment]
Hello, just looking for answers to the above. Let me know,, thanks.

**Comment by gonchik** [comment]
Hello, 
Could you install that extension to your browser
https://www.editthiscookie.com/


 then copy into file and follow instruction on main file of instruction please
https://docs.python.org/3/library/http.cookiejar.html

**Comment by eabase** [comment]
Can we have some update on this issue please?
How can we use this with a Jira or Confluence API token?

**Comment by gonchik** [comment]
Doc has been delivered. forgot to follow up
